src/controllers/controllerscreenpreview.cpp

```diff
@@ -0,0 +1,60 @@
+#include "controllers/controllerscreenpreview.h"
+
+#include <QHBoxLayout>
+#include <QVBoxLayout>
+
+#include "moc_controllerscreenpreview.cpp"
+#include "util/time.h"
+
+ControllerScreenPreview::ControllerScreenPreview(
```

Comment:
please add doc comments to all new classes

Reply:
I have already put comments in the headers file, I'm going to extends it as they are quite short. WOuld you like me to duplicate them in the cpp file too?

Reply:
nope, .h is enough, thanks!